services/frontend/src/components/resultsCard/resultProfileCard/ResultProfileCardView.jsx

```diff
@@ -202,7 +202,7 @@ const ResultProfileCardView = ({
     </div>,
     <div>
       <EnvironmentOutlined className="result-card-footer-icon" />
-      {user.branch ? (
+      {profile.officeLocatio ? (
```

Comment:
should that be officeLocation ?

Reply:
omg i can believe what I just committed lol